hazelcast/src/main/java/com/hazelcast/config/NamespaceConfig.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.config;
+
+import com.hazelcast.internal.config.ConfigDataSerializerHook;
+import com.hazelcast.internal.namespace.ResourceDefinition;
+import com.hazelcast.internal.namespace.impl.ResourceDefinitionImpl;
+import com.hazelcast.internal.serialization.impl.SerializationUtil;
+import com.hazelcast.jet.config.ResourceConfig;
+import com.hazelcast.jet.config.ResourceType;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class NamespaceConfig implements NamedConfig, IdentifiedDataSerializable {
```

Comment:
Addressed in: https://github.com/vbekiaris/hazelcast/commit/3ba9010e5b891308dd4c808cac78075d54391a8c

I've been using `@since 5.4` and looking around the codebase there's a mix of `X.Y` and `X.Y.Z` declarations - any preference here? Worth updating to match?